nautobot/utilities/forms/forms.py

```diff
@@ -32,9 +32,10 @@ def __init__(self, *args, **kwargs):
         instance = kwargs.get("instance")
         initial = kwargs.get("initial", {}).copy()
 
+        # If we're creating a new ip from prefix, initial address will be filled in.
         # If we're editing an object with a `address` field, we need to patch initial to include
         # `address` because it is a computed field.
-        if instance is not None:
+        if initial.get("address", None) is None and instance is not None:
```

Comment:
Is the `"address"` key entirely absent in the generic case? Could we simplify this as follows and will it still work?

```suggestion
        if "address" not in initial and instance is not None:
```

If that doesn't work, then perhaps:

```suggestion
        if not initial.get("address") and instance is not None:
```

In either case, same suggestion below.

Reply:
Let me test and get back to you.  I originally had "if not initial.get("address")" but a colleague recommended the path I went.